docker/data-science/notebooks/ch05/3.semantic-knowledge-graph.ipynb

```diff
@@ -664,48 +652,27 @@
       "impala  0.45309\n",
       "streamsets  0.39341\n",
       "scala  0.38564\n",
-      "flume  0.38401\n",
-      "attunity  0.37374\n",
-      "mapreduce  0.36195\n"
+      "flume  0.38401\n"
```

Comment:
TODO: check this output vs. manuscript. I don't think this example is in the manuscript (so no problem if not).

Reply:
min_popularity